OsmAnd/res/values/strings.xml

```diff
@@ -543,9 +543,9 @@
     <string name="map_widget_magnetic_bearing">Magnetic bearing</string>
     <string name="map_widget_bearing">Relative bearing</string>
     <string name="access_disable_offroute_recalc">No route recalculation after leaving it</string>
-    <string name="access_disable_offroute_recalc_descr">Keeps the same route when you go offroad.</string>
+    <string name="access_disable_offroute_recalc_descr">No automatic route recalculation after going off route.</string>
     <string name="access_disable_wrong_direction_recalc">No route recalculation for opposite direction</string>
-    <string name="access_disable_wrong_direction_recalc_descr">Keeps the same route when you move in the fully wrong direction.</string>
+    <string name="access_disable_wrong_direction_recalc_descr">Prevents automatic route recalculation while just moving in the opposite direction.</string>
```

Comment:
"Preventing automatic route recalculation while" is more complex than
"Keeps the same route when"